src/main/java/jenkins/plugins/gerrit/client/ssh/GerritClientSSH.java

```diff
@@ -0,0 +1,188 @@
+package jenkins.plugins.gerrit.client.ssh;
+
+import com.google.common.base.Ascii;
+import com.google.gerrit.extensions.api.changes.*;
+import com.google.gerrit.extensions.common.ChangeInfo;
+import com.google.gerrit.extensions.restapi.RestApiException;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.jcraft.jsch.Session;
+import com.urswolfer.gerrit.client.rest.http.changes.ChangesParser;
+import com.urswolfer.gerrit.client.rest.http.changes.ReviewResultParser;
+import jenkins.plugins.gerrit.client.GerritClient;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jgit.transport.RemoteSession;

```

Comment:
interesting, in my mind I would have sent commands using low level ssh library such as apache-sshd instead of using jgit. but maybe this is better.

Reply:
My reasoning was that I should use whatever Jenkins uses and if I recall correctly Jenkins uses Trilead, which I believe is somewhat intertwined with Jgit.